packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -749,11 +749,12 @@ const DropDownEditor = TextBox.inherit({
     },
 
     _getFirstPopupElement: function() {
-        return this._popup.$wrapper().find('.dx-popup-done.dx-button');
+        return $(this._popup.getAllElements()[0]);
     },
 
     _getLastPopupElement: function() {
-        return this._popup.$wrapper().find('.dx-popup-cancel.dx-button');
+        const elements = this._popup.getAllElements();
+        return $(elements[elements.length - 1]);
```

Comment:
I don't see changed in ddButton. It's not an inheritor of ddEditor, but has the same functionality. Please, check it

Reply:
Apparently ddButon does not have such functionality. There is no the `applyValueMode` in the documentation and dropDownOptions.toolbarItems is not applied. The code also no such options. [Doc](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#dropDownOptions) is not tell something about it. I think it can be forgot to put in the doc

Reply:
ok, thanks for check